### PR TITLE
Cartbounty bug fix for variant products

### PR DESCRIPTION
**Pull request by angbpy**

### CODE REVIEW

**Comment by ChetanGN** [comment]
@angbpy we should not invoke `handleCBRecoveredOrder` in case of native checkout. Please add the condition on the callback function.

**Comment by angbpy** [comment]
> @angbpy we should not invoke `handleCBRecoveredOrder` in case of native checkout. Please add the condition on the callback function.

Updating by checking the 1cc condition

File: woo-razorpay.php [hunk 1]
```diff
@@ -1552,7 +1552,9 @@ public function updateOrder(& $order, $success, $errorMessage, $razorpayPaymentI
                     $order->payment_complete($razorpayPaymentId);
                 }
 
-                if(is_plugin_active('woo-save-abandoned-carts/cartbounty-abandoned-carts.php')){
+                $is1ccOrder = get_post_meta( $wcOrderId, 'is_magic_checkout_order', true );
```
Comment (ChetanGN) [inline woo-razorpay.php:1555]: @angbpy Why do we need to get the post meta info here?

Comment (angbpy) [reply woo-razorpay.php:1555]: We are receiving the info whether the order that is placed is a 1cc order or not from the post meta table

Comment (ChetanGN) [reply woo-razorpay.php:1555]: @angbpy It's already defined in this [line](https://github.com/razorpay/razorpay-woocommerce/blob/736fbb04c7753c01ec503e77a9b6224bba7814a4/woo-razorpay.php#L1521), can we utilize the same?

Comment (angbpy) [reply woo-razorpay.php:1555]: In case of an error in the try block will it still reach this line or then we shouldn't worry about it?

**Review by ChetanGN — APPROVED** [review]

**Pull request merged**